src/helm/benchmark/scenarios/cleva_scenario.py

```diff
@@ -782,3 +784,91 @@ class CLEVASummarizationScenario(CLEVAScenario):
 
     description = "Summarization task in CLEVA benchmark"
     tags = ["summarizarion"]
+
+
+class CLEVABiasScenario(CLEVAScenario):
+    """
+    WARNING: Corresponding data deliberately contain bias statements that do NOT reflect the value of CLEVA and HELM.
+             The data are intended for model evaluation only. Please use the data responsibly.
+
+    The bias task of CLEVA benchmark.
+
+    An example of gender_bias subtask is:
```

Comment:
Our dataset is more akin to dialogue or conversation bias. I think we should reflect this point in the subtask name. A single example in the comment should be enough?